web/styles/reactions.css

```diff
@@ -73,10 +73,13 @@
         background-color: hsl(0deg 0% 98%);
     }
 
-    .reaction_button {
+    .reaction_button,
+    .emoji_message_control_button_container {
```

Comment:
Should this new class name use dashes?

Reply:
Yeah, sure! Changed the underscores to dashes.